tide_core.install

```diff
@@ -1548,3 +1548,29 @@ function tide_core_update_8064() {
     }
   }
 }
+
+/**
+ * Add contact field to user account(business name, phone, notes).
+ */
+function tide_core_update_8065() {
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_location = [drupal_get_path('module', 'tide_core') . '/config/install'];
```

Comment:
this is deprecated from drupal/core 9.3 - `drupal_get_path('module',`
Please use the following one as from the next release we are updating every project to 9.4 - 
`$config_location = [\Drupal::service('extension.list.module')->getPath`
See this - https://www.drupal.org/node/2940438